### PR TITLE
chore: remove unused dependencies

### PR DESCRIPTION
**Pull request by flotwig**
- Closes 

### Additional details

- Used depcheck to check some of our packages for unused packages.

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/ui-components/package.json [hunk 1]
```diff
@@ -28,15 +28,13 @@
...
     "mobx": "5.15.4",
     "mobx-react": "6.1.7",
     "prop-types": "15.7.2",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "webpack": "4.41.6",
-    "webpack-cli": "3.3.11"
+    "webpack": "4.41.6"
   }
 }
```
Comment (emilyrohrbough) [inline packages/ui-components/package.json:40]:
```suggestion
}

```

File: packages/driver/cypress/fixtures/dropdown.html [hunk 2]
```diff
@@ -1,44 +0,0 @@
-<!DOCTYPE html>
```
Comment (emilyrohrbough) [inline packages/driver/cypress/fixtures/dropdown.html:1]: Did we lose test coverage at some point ? did you look into when these became obsolete?

**Review by lmiller1990 — COMMENTED** [review]
Cool. How did you figure these were unused? Did you use some kind of tool to do the analysis?

Seems like something we should be doing on a schedule, or possibly automatically.

**Comment by flotwig** [comment]
@lmiller1990 It's a npm module called `depcheck`, it works okay. I think eventually we should add it to our `lint` step in CI so we can really button this up. It also notices when dependencies are referenced but not defined in `package.json`.

File: packages/driver/cypress/fixtures/dropdown.html (hunk 2 above)
Comment (flotwig) [reply packages/driver/cypress/fixtures/dropdown.html:1]: Maybe my Git-fu is weak, but I was unable to tell when these files were ever being used for testing. They are over 5 years old.

**Review by ryanthemanuel — COMMENTED** [review]
Nice work! Always nice to clean things up.

**Review by ryanthemanuel — APPROVED** [review]

**Review by emilyrohrbough — APPROVED** [review]
I spot checked quite a few of these and everything appears to be jusssst fine. Thanks for doing this! I'm always happy to remove dependencies 😄

**Pull request merged**